bundles/org.openhab.binding.insteon/src/main/java/org/openhab/binding/insteon/internal/handler/InsteonDeviceHandler.java

```diff
@@ -371,7 +372,11 @@ public void initialize() {
                     }
                 });
 
-                updateStatus(ThingStatus.ONLINE);
+                if (ThingStatus.ONLINE.equals(bridge.getStatus())) {
```

Comment:
Simplification possible:
```suggestion
                if (ThingStatus.ONLINE == bridge.getStatus()) {
```

Reply:
changed even though it really doesn't matter

Reply:
It was suggested as a possibility, not a requirement. Argument for using `==` over `equals` on enum:
- It's faster.
- It's safer at run-time.
- It's safer at compile-time.